SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -1,8 +1,12 @@
 import Cocoa
 
-class AppCenterViewController : NSViewController {
+// 100 MiB.
+let kMSDefaultDatabaseSize = 100 * 1024 * 1024
```

Comment:
Let's use the same default as other SDKs - 10MiB

Reply:
Fix in new commit.